CMP1903_A1_2324/Program.cs

```diff
@@ -14,6 +14,29 @@ static void Main(string[] args)
              * Create a Game object and call its methods.
              * Create a Testing object to verify the output and operation of the other classes.
              */
+
+            /// <summary>
+            /// Infinitely creates a game object through the Testing class
+            /// until the users types 'stop'.
+            /// </summary>
+
+
+            // Keep rolling until 'stop' is typed
+            for (; ; )
```

Comment:
A use of a indefinite while loop may be a better choice compared to a definite for loop.